clang/test/dpct/cmake_migation/case_004/expected.txt

```diff
@@ -0,0 +1,14 @@
+target_link_libraries(main PRIVATE cxxopts fmt::fmt -qmkl)
+
+target_link_libraries(transformer_engine PUBLIC 
+   -qmkl
+
+
+
+
+   libdnn.so)
```

Comment:
For Linux, we need use `-dnnl`; for Windows, we need use `dnnl.lib`

Reply:
To differentiate the mapping for Linux and Windows, it is better to introduce a cmake helper function. You can ask @TejaX-Alaghari for more detail.

Reply:
Added in seperate PR: https://github.com/oneapi-src/SYCLomatic/pull/1544